### DISCUSSION TITLE
why january 05 2024 including "de S. Maria (vel ad libitum)"?

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#4211] [Q&A]
Hi I have a question about this one, i could not understand why is including oratio de S. Maria (vel ad libitum). 
Thanks

**Comment by FAJ-Munich** [comment]
Which Rubrics are you referring to?

**Reply by d4h0y0sr** [reply]
1960

**Reply by FAJ-Munich** [reply]
should be solved as well after PR #4215 is merged
Let me know if not.
